lib/stencil-bundle.js

```diff
@@ -288,6 +306,10 @@ function bundleTaskRunner(callback) {
         defaultName = 'Theme.zip';
     }
 
+    const distExists = this.distCheck();
```

Comment:
I'm surprised that the dist files exist at this point in time. They shouldn't be generated until the tasks are run below, specifically `tasks.theme`. Instead of putting this here, you should put it inside the callback for `Async.parallel` below.